oidc-provider/simple_op/src/provider/authn/__init__.py

```diff
@@ -0,0 +1,30 @@
+import importlib
```

Comment:
It would be good to add our usual imports to the top of any significant (that is, non-init) python file:

```
from __future__ import division
from __future__ import print_function
from __future__ import unicode_literals
```